secrets.yml

```diff
@@ -1,15 +1,18 @@
+SWIFT_TENANT: 'tester'
```

Comment:
Do we want this file to be committed into the codebase? 
maybe secrets_example.yml is sufficient?

Reply:
Will remove. No longer needed as I've changed the tests and allowed the command-line specification of a config file - nice catch!